ckanext/googleanalytics/controller.py

```diff
@@ -16,6 +16,9 @@
 
 from ckan.controllers.api import ApiController
 from ckan.controllers.package import PackageController
+from ckan.common import g
+if 'cloudstorage' in g.plugins:
```

Comment:
I would set a global `is_using_cloudstorage` here and check that at line 155 instead of testing `g.plugins` again.